src/app/package-json.js

```diff
@@ -86,6 +86,7 @@ const packageJSON = (current, context) => {
     'version': context.version,
     'description': context.description,
     'main': 'es5/plugin.js',
+    'browser': 'src/plugin.js',
```

Comment:
Yes, it's a two-step process because we need both the transpiled code (in `es5/`) and the Browserified output (in `dist/`).